src/Farmer/Builders.ContainerRegistry.fs

```diff
@@ -0,0 +1,61 @@
+[<AutoOpen>]
+module Farmer.Resources.ContainerRegistry
+
+open Farmer
+open Farmer.Models
+
+/// Container Registry SKU
+type ContainerRegistrySku =
+    | Basic
+    | Standard
+    | Premium
+
+// TODO: networkRuleSet
+// TODO: policies
+// TODO: encryption
+// TODO: dataEndpointEnabled
+
+type ContainerRegistryConfig =
+    {
+      Name : ResourceName
+      Sku : ContainerRegistrySku
+      AdminUserEnabled : bool }
+    
+type ContainerRegistryBuilder() =
+    member _.Yield _ =
+        { Name = ResourceName.Empty
+          Sku = Basic
+          AdminUserEnabled = false }
+    [<CustomOperation "name">]
```

Comment:
Could you put /// comments on here as well - I *believe* that these feed through into client code.

Reply:
@isaacabraham I added the comments